pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1469,16 +1512,20 @@ func TestPreFilterState(t *testing.T) {
 			p.(*PodTopologySpread).enableMatchLabelKeysInPodTopologySpread = tt.enableMatchLabelKeys
 
 			cs := framework.NewCycleState()
-			if _, s := p.(*PodTopologySpread).PreFilter(ctx, cs, tt.pod); !s.IsSuccess() {
-				t.Fatal(s.AsError())
-			}
-			got, err := getPreFilterState(cs)
-			if err != nil {
-				t.Fatal(err)
-			}
-			if diff := cmp.Diff(tt.want, got, cmpOpts...); diff != "" {
+			_, s := p.(*PodTopologySpread).PreFilter(ctx, cs, tt.pod)
+			if diff := cmp.Diff(tt.wantPrefilterStatus, s, cmpOpts...); diff != "" {
```

Comment:
My bad, I've just found a more appropriate one.

```suggestion
			if !tt.wantPrefilterStatus.Equal(s) 
```

Then, you shouldn't need to put `wantPrefilterStatus: framework.NewStatus(framework.Success),` in each test case.
(nil is equivalent to framework.NewStatus(framework.Success))